e2etest/bgp_test.go

```diff
@@ -93,32 +93,42 @@ var _ = ginkgo.Describe("BGP", func() {
 			},
 		},
 	}
-	ginkgo.BeforeEach(func() {
-		var err error
-		if containersNetwork == "host" {
-			if net.ParseIP(hostIPv4) == nil {
-				framework.Fail("Invalid hostIPv4")
-			}
-			if net.ParseIP(hostIPv6) == nil {
-				framework.Fail("Invalid hostIPv6")
+
+	ginkgo.AfterEach(func() {
+		if ginkgo.CurrentGinkgoTestDescription().Failed {
+			for _, c := range frrContainers {
+				dump, err := frr.RawDump(c)
```

Comment:
Can you have this dumped to a file instead/in addition? I think it'll be easier to navigate a file (that is also uploaded to the ci's artifacts).

Reply:
I am torn about this. It found dumping it as part of the log execution made quick debugging easier even without the need to download the files because you see the failed test and immediately go above and see the configuration.
Having it in a file means we need to download, cross check, etc. I'd keep it as it is for now.